alt-fuel-finder/app/services/nrel_service.rb

```diff
@@ -1,26 +1,24 @@
 class NrelService < OpenStruct
 
   def initialize
-
     @conn = Faraday.new("https://developer.nrel.gov")
   end
 
-  def nearest_stations_by_zip(zip)
-    raw = conn.get("/api/alt-fuel-stations/v1/nearest.json?api_key=S9nWjdAxKadPGBVKfBnjdIRaemILUjbVKpY5M6Zi&location=#{zip}&radius=6&fuel_type=ELEC,LPG")
+  def nearest_stations_by_zip(zip, limit = 10)
+    raw = conn.get("/api/alt-fuel-stations/v1/nearest.json",
+                  { api_key: "S9nWjdAxKadPGBVKfBnjdIRaemILUjbVKpY5M6Zi",
+                    location: zip,
+                    radius: 6,
+                    fuel_type: "ELEC,LPG",
+                    limit: limit
+                  })
```

Comment:
bah! Forgot to remove  the OpenStruct inheritance. Obviously, it should just be an `NrelService` class, shouldn't inherit from anything.